doc/nrf/conf.py

```diff
@@ -192,7 +192,15 @@
     "latest" if version.endswith("99") else version
 )
 
+
 def setup(app):
     app.add_css_file("css/nrf.css")
 
+    app.add_js_file(
+        "https://policy.app.cookieinformation.com/uc.js",
+        id="CookieConsent",
+        type="text/javascript",
+        **{"data-culture": "EN"},
+    )
```

Comment:
should be put inside `add_google_analytics`

Reply:
Absolutely, fixed